staging/src/k8s.io/client-go/rest/with_retry.go

```diff
@@ -179,10 +179,8 @@ func (r *withRetry) prepareForNextRetry(ctx context.Context, request *Request) e
 
 	// Ensure the response body is fully read and closed before
 	// we reconnect, so that we reuse the same TCP connection.
-	if seeker, ok := request.body.(io.Seeker); ok && request.body != nil {
-		if _, err := seeker.Seek(0, 0); err != nil {
-			return fmt.Errorf("can't Seek() back to beginning of body for %T", request)
-		}
+	if request.body != nil {
```

Comment:
I don't think the comment was correct. My understanding is that we're trying to reset the body to prep it for the retry. Discarding the body content isn't what we want

Reply:
kind of rewinding, right?

Reply:
Right